cmd/createorupdate/createorupdate.go

```diff
@@ -121,16 +120,16 @@ func enrich(cs *acsapi.ContainerService) error {
 		return fmt.Errorf("must set RESOURCEGROUP")
 	}
 
+	// TODO: MS to provide FQDN values for OCP cluster to use.
```

Comment:
Shouldn't you simulate these steps in `hack/create.sh`?

Reply:
yes. But all enrich function should be gone either way. Just choosing my tool until we agree on the right way.